scripts/cms5/phpstan.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 // Only valid for non-theme modules
-if (!is_module() || is_theme() || module_is_recipe()) {
+if (!is_module() || is_theme() || module_is_recipe() || is_tooling()) {
```

Comment:
Tooling is by definition not a module, why do we need this extra part of the condition?

Reply:
behat-extension is both is_module() and is_tooling()

Reply:
How? Behat extension has the `behat-extension` type in its composer.json.

Reply:
opps, sorry meant frameworktest

Reply:
Hmmm that's an interesting point.
I tried to think of the implications and couldn't think of this causing us any problems... I _think_ this is probably okay. 👍